x-pack/platform/plugins/shared/alerting/server/plugin.ts

```diff
@@ -343,6 +345,42 @@ export class AlertingPlugin {
     });
     this.ruleTypeRegistry = ruleTypeRegistry;
 
+    plugins.taskManager.registerTaskDefinitions({
+      taskWithApiKey: {
+        title: 'taskWithApiKey',
+        createTaskRunner: ({ taskInstance }) => ({
+          run: async () => {
+            const services = await core.getStartServices();
+
+            const fakeRawRequest: FakeRawRequest = {
```

Comment:
This seems like a lot of boilerplate for clients of this functionality.  Feels like we should at least give them a FakeRequest already to go, assuming they are lightweight.  I could imagine later giving folks es/so clients scoped to the request.